ansible/roles/test/tasks/single_lag_test.yml

```diff
@@ -0,0 +1,97 @@
+### Part of lag test palybook lag_2.yml (--tag lag_2)
+### This playbook test one single port channel minimum link feature of one member interface shutdown 
+### and portchannel member interfaces sending ACP DU rate
+
+# Gather information of port channel ports, minimum links and total interface member numbers
+- set_fact: 
+    po: "{{ item }}"
+    po_interfaces: "{{ lag_facts.lags[item]['po_config']['ports'] }}"
+    po_intf_num: "{{ lag_facts.lags[item]['po_config']['ports']|length }}"
+    po_min_links: "{{lag_facts.lags[item]['po_config']['runner']['min_ports']}}"
+
+# pick flap interface name and calculate when it flaps, should portchannel interface flap or not
+# Current it is using a static capacity < 75%, Portchannel will flap which match Sonic configuration
+# if need to be random, then will make it a var
+- set_fact:
+    po_flap: "{{ (po_intf_num|float - 1)/(po_min_links|float)*100 < 75 }}"
+    flap_intf: "{{ lag_facts.lags[item]['po_config']['ports'].keys()[0] }}"
+
+### figure out fanout switches info for the flapping lag member and run minlink test
+- set_fact:
+    peer_device: "{{ fanout_neighbors[flap_intf]['peerdevice'] }}"
+    neighbor_interface: "{{ fanout_neighbors[flap_intf]['peerport'] }}"
+
+- conn_graph_facts: host={{ peer_device }}
+  connection: local
+
+- set_fact:
+    peer_host: "{{ device_info['mgmtip'] }}"
+    peer_hwsku: "{{ device_info['HwSku'] }}"
+
+- name: test fanout interface (physical) flap and lacp keep correct po status follow minimum links requirement
+  include: lag_minlink.yml
+  vars:
+    wait_down_time: 20
+
+### Now figure out remote VM and interface info for the falpping lag member and run minlink test 
+- set_fact:
+    peer_device: "{{vm_neighbors[flap_intf]['name']}}"
+    neighbor_interface: "{{vm_neighbors[flap_intf]['port']}}"
+    peer_hwsku: 'Arista-VM'
+
+- set_fact:
+    peer_host: "{{ lldp[flap_intf]['chassis']['mgmt-ip'] }}"
```

Comment:
please mark this as todo, we should really use the testbed info to deduce the mgmt ip of the arista vm, not using lldp.